autoload/lsp/lspserver.vim

```diff
@@ -832,7 +832,8 @@ def ShowHoverInfo(lspserver: dict<any>): void
   # interface HoverParams
   #   interface TextDocumentPositionParams
   var params = GetLspTextDocPosition(false)
-  lspserver.rpc_a('textDocument/hover', params, hover.HoverReply)
+  lspserver.rpc_a('textDocument/hover', params,
+			function(hover.HoverReply, [noEcho]))
```

Comment:
I assume that you can use a lambda instead:

```
lspserver.rpc_a('textDocument/hover', params, (lspserver, reply) => {
    hover.HoverReply(lspserver, reply, noEcho)
})
```

if you want `noEcho` to be the last argument instead of the first. I think it makes sense to strive to always have the `lspserver` and `reply` as the first arguments

Reply:
thx for tips. 
but i am not plan to refactor this code, i just kept the same code style like others in this repo.
// maybe we can refactor it later.